src/Uno.UI/UI/Xaml/Media/ImageSource.iOS.cs

```diff
@@ -152,8 +155,7 @@ internal async Task<UIImage> Open(CancellationToken ct)
 
 					if (SupportsAsyncFromBundle)
 					{
-						// Since iOS9, UIImage.FromBundle is thread safe, so we need to disable 
-						// the Xamarin binding check, which is unconditional.
+						// Since iOS9, UIImage.FromBundle is thread safe.
```

Comment:
True, but only in release mode. This will fail in debug (the calls to UIApplication.CheckForIllegalCrossThreadCalls are made regardless of the thread safety-ness of the API)